compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -206,13 +206,14 @@ static void addBufferizePasses(OpPassManager &funcPassManager) {
 static void
 tileAndDistributeToWorkgroup(OpPassManager &funcPassManager,
                              bool useWARForCooperativeMatrixCodegen = false,
-                             bool convertInputsToDestinations = true) {
+                             bool convertInputsToDestinations = true,
```

Comment:
Eh, maybe just pass in `std::optional<ConvertToDestinationPassingStylePassOptions> convertToDpsOptions = ConvertToDpsOptions{}` (assuming that this sets the values to default.

Reply:
Good idea, this looks cleaner